### DISCUSSION TITLE
Why PopUp is blocked with google auth?

### DISCUSSION THREAD

**Discussion by Josutoral** [capawesome-team/capacitor-firebase#349] [Q&A]
I'm implementing authentication with google for my web app. I can select the google email and receive the data but then the pop up closes and I get this error again: 'Popup blocked'. How can I fix?

**Comment by robingenz** [comment]
This is far too little information to help you. 
Please provide a Minimal, Reproducible Example (see [How to create a Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)) in a public GitHub repository. 
Please also share details like which platform you use, which device, which versions, ...
